ts/model_service_worker.py

```diff
@@ -22,7 +21,10 @@
 DEBUG = False
 BENCHMARK = os.getenv("TS_BENCHMARK")
 BENCHMARK = BENCHMARK in ["True", "true", "TRUE"]
-
+LOCAL_RANK = int(os.getenv('LOCAL_RANK', 0))
```

Comment:
by default i don't believe these environment variables will be set in a single GPU deployment